.bumpversion.cfg

```diff
@@ -2,5 +2,3 @@
 current_version = 6.21.0
 commit = True
 tag = True
-
-[bumpversion:file:kolibri_explore_plugin/__init__.py]
```

Comment:
I overall love this, but what does bumpversion do after this? I suppose that it just makes the commit updating `current_version` in this file and tags with the desired version.

Reply:
> I suppose that it just makes the commit updating current_version in this file and tags with the desired version.

Yes, exactly